tutorials/create-serverless-scraping/index.mdx

```diff
@@ -0,0 +1,361 @@
+    ```
+    resource "scaleway_function_cron" "scraper_cron" {
+      function_id = scaleway_function.scraper.id 
+      schedule = "0,15,30,45 * * * *"
+      args = jsonencode({})
+    }
+
+    resource "scaleway_function_trigger" "consumer_sqs_trigger" {
+      function_id = scaleway_function.consumer.id 
+      name = "hn-sqs-trigger"
+      sqs {
+        project_id = scaleway_mnq_sqs.main.project_id
+        queue = scaleway_mnq_sqs_queue.main.name
+      }
+    }
+    ```
+
+## Put it all in motion
+Terraform makes this very straightforward. To provision all the resources and get everything up and running, run the following commands:
+```
+cd terraform
+terraform init
+terraform plan 
+terraform apply
+```
+
+### How to check that everything is working correctly
+
+First, go to the [Scaleway console](https://console.scaleway.com/), [activate Cockpit](/observability/cockpit/quickstart/), and check the logs and metrics for Serverless Functions' execution and Messaging and Queuing SQS queue statistics.
+
+To check whether the data is correctly stored in the database, you can try [connecting to it directly](/managed-databases/postgresql-and-mysql/how-to/connect-database-instance/) via a CLI tool such as `psql`. 
+Retrieve the instance IP and port of your Managed Database from the console, under the [Managed Database section](https://console.scaleway.com/rdb/instances).
+Use the following command to connect to your database. When prompted for password, you can find it by running `terraform output -json`.
+```
+psql -h <DB_INSTANCE_IP> --port <DB_INSTANCE_PORT> -d hn-database -U worker
+```
+
+When you are done testing, don't forget to clean up! First deactivate the cockpit from the console, if you had activated it. Then run: 
+```
+cd terraform
+terraform destroy
+```
+
+## Summary, going further, key takeaways
+We have showed how to asynchronously decouple producer and consumer using SQS, adhering to serverless patterns. 
+While the volume of data processed in this example is not really noteworthy, thanks to the Messaging and Queueing SQS queue's robustness and the auto-scaling capabilities of the Serverless Functions, you can adapt this example to manage larger workloads.
+
+Here are some possible extensions to this basic example:
+ - Replace the simple proposed logic with your own. What about counting how many times some keywords (e.g: copilot, serverless, microservice) appear in Hacker News articles? 
+ - Define multiple cron triggers for different websites and pass the website as an argument to the function. Or, create multiple functions that feed the same queue. 
+ - Use a serverless container instead of the consumer function, and use a command line tool such as `htmldoc` or `pandoc` to convert the article to PDF and upload the result to a [Scaleway Object Storage](/storage/object/quickstart/) S3 bucket. 
```

Comment:
```suggestion
 - Use a [Serverless Container](/serverless/containers/quickstart/) instead of the consumer function, and use a command line tool such as `htmldoc` or `pandoc` to convert the scraped articles to PDF and upload the result to a [Scaleway Object Storage](/storage/object/quickstart/) S3 bucket. 
```